subsys/bluetooth/shell/has.c

```diff
@@ -75,6 +75,43 @@ static int cmd_preset_unreg(const struct shell *sh, size_t argc, char **argv)
 	return 0;
 }
 
+static int cmd_has_register(const struct shell *sh, size_t argc, char **argv)
+{
+	int err;
+	struct bt_has_register_param param = {
+		.type = BT_HAS_HEARING_AID_TYPE_MONAURAL,
+		.preset_sync_support = false,
+		.independent_presets = false
+	};
+
+	for (size_t argn = 1; argn < argc; argn++) {
+		const char *arg = argv[argn];
+
+		if (strcmp(arg, "binaural") == 0) {
```

Comment:
```suggestion
		if (strcmp(argv[1], "binaural") == 0) {
```
Sorry, a small detail, but I would prefer it close to this (and adjust the rest accordingly), to avoid several type inputs. This seems also like how its been done in other places for shell

Reply:
Seems there are both approaches already. Personally Id prefer locked positions for the arguments, but I dont think it should stand in the way of this now.